bot.js

```diff
@@ -75,29 +75,40 @@ client.on('message', msg => {
         msg.channel.send(`${days} days, ${hrs} hours, ${minutes} minutes and ${Math.floor(seconds)} seconds left! IT TAKES FOREVERHHH`);
     }
 
-    if (msg.content.toLowerCase().includes('who is best pony')) {
-        if (talkedRecently.has(msg.channel.id + bestPonyType)) {
-            sendCooldownMessage(msg, bestPonyType);
-        } else {
-            msg.channel.send(msg.author + ` ${getBizaamEmoji()} I am, of course!`);
-
-            talkedRecently.add(msg.channel.id + bestPonyType);
-            setTimeout(() => {
-              talkedRecently.delete(msg.channel.id + bestPonyType);
-            }, 60000);
-        }
-    }
-
-    if (msg.content.toLowerCase().includes(' is best pony') && !msg.content.toLowerCase().includes('who is best pony')) {
-        if (talkedRecently.has(msg.channel.id + interjectType)) {
-            sendCooldownMessage(msg, interjectType);
+    if (msg.content.toLowerCase().includes(' is best pony')) {
+        if (msg.content.toLowerCase().includes('who is best pony')){
+            if (talkedRecently.has(msg.channel.id + bestPonyType)) {
+                sendCooldownMessage(msg, bestPonyType);
+            } else {
+                msg.channel.send(msg.author + ` ${getBizaamEmoji()} I am, of course!`);
+
+                talkedRecently.add(msg.channel.id + bestPonyType);
+                setTimeout(() => {
+                  talkedRecently.delete(msg.channel.id + bestPonyType);
+                }, 60000);
+            }
+        } else if (msg.content.toLowerCase().includes('canni is best pony')) {
+              if (talkedRecently.has(msg.channel.id + bestPonyType)) {
```

Comment:
`bestPonyType` should be replaced with a unique type for this message. Otherwise the timeout/CD will conflict with the **who is best pony** command.